mutt-ical.py

```diff
@@ -147,7 +147,10 @@ def display(ical):
                 sys.stdout.write(attendee.CN_param + " <" + attendee.value.split(':')[1] + ">, ") #workaround for MS
             except: 
                 sys.stdout.write(attendee.value.split(':')[1] + " <" + attendee.value.split(':')[1] + ">, ") #workaround for 'mailto:' in email
-    sys.stdout.write("\n\n")
+    sys.stdout.write("\n")
+    print("Start:\t%s" % (ical.vevent.dtstart.value,))
```

Comment:
Are `dtstart` and `dtend` never `None`?

Reply:
I've updated the pull request to allow for missing attributes.